src/accounts/models.py

```diff
@@ -69,3 +70,10 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.unique_identifier} as {self.username}"
+
+class PasswordResetRequestModel(models.Model):
+    token = models.TextField()
+    account = models.ForeignKey(Account, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+    def is_token_valid(self):
+        return timezone.now() <= self.created_at + 30
```

Comment:
this implementation doesn't look right. It should be something like
```suggestion
        return timezone.now() <= timedelta(minutes=30)
```

Although I suggested on Discord to make this an hour instead.

We should probably include a command as well, so we can periodically look for dead tokens and remove them from the database.